rhel.Dockerfile

```diff
@@ -0,0 +1,129 @@
+          pushd ${d}/lib/python3.6/site-packages/ >/dev/null; \
+            cp -r PyYAML* xmltodict* yaml* yq* jsonschema* /usr/lib/python3.6/site-packages/; \
+          popd >/dev/null; \
+        fi; \
+      done; \
+      chmod -c +x /usr/local/bin/*; \
+    else \
+      /usr/bin/pip3.6 install yq jsonschema; \
+    fi && \
+    ln -s /usr/bin/python3.6 /usr/bin/python && \
+    # test install worked
+    for d in python yq jq jsonschema; do echo -n "$d: "; $d --version; done
+
+# for debugging only
+# RUN microdnf install -y util-linux && whereis python pip jq yq && python --version && jq --version && yq --version
+
+################# 
+# PHASE TWO: configure registry image
+#################
+
+COPY ./scripts/*.sh ./scripts/meta.yaml.schema /build/
+COPY /v3 /build/v3
+WORKDIR /build/
+
+# if only including the /latest/ plugins, apply this line to remove them from builder
+# RUN rm -fr $(find /build/v3 -name 'meta.yaml' | grep -v "/latest/" | grep -o ".*/")
+
+# not supported in Brew unless we prefetch the content via tarball injection
+# optional steps for air gap - replace references to docker.io, quay.io, registry.access.redhat.com, registry.redhat.io with internal registry
+# RUN ./list_containers.sh v3 && ./replace_container_repos.sh v3 myquay.mycorp.com
+
+RUN ./check_plugins_location.sh v3 && \
+    ./set_plugin_dates.sh v3 && \
+    ./check_plugins_viewer_mandatory_fields.sh v3 && \
+    ./ensure_latest_exists.sh && \
+    ./index.sh v3 > /build/v3/plugins/index.json && \
+    chmod -c -R g+rwX /build
+
+################# 
+# PHASE THREE: create ubi8-minimal image with httpd
+################# 
+
+# Build registry, copying meta.yamls and index.json from builder
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhscl/httpd-24-rhel7
+FROM registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-104 AS registry
+USER 0
+
+# BEGIN these steps might not be required
+RUN sed -i /etc/httpd/conf/httpd.conf \
+    -e "s,Listen 80,Listen 8080," \
+    -e "s,logs/error_log,/dev/stderr," \
+    -e "s,logs/access_log,/dev/stdout," \
+    -e "s,AllowOverride None,AllowOverride All," && \
+    chmod a+rwX /etc/httpd/conf /run/httpd
+STOPSIGNAL SIGWINCH
+# END these steps might not be required
+
+COPY README.md .htaccess /var/www/html/
+COPY --from=builder /build/v3 /var/www/html/v3
+COPY ./scripts/*entrypoint.sh /usr/local/bin/
+
+WORKDIR /var/www/html
+ENTRYPOINT ["/usr/local/bin/uid_entrypoint.sh", "/usr/local/bin/entrypoint.sh"]
+
+# Offline build: cache .theia and .vsix files in registry itself and update metas
+# multiple temp stages does not work in Brew
+FROM builder AS offline-builder
+
+# To only cache files from /latest/ folders, use ./cache_artifacts.sh v3 --latest-only 
+# and uncomment line above to remove files so they're not included in index.json -- RUN rm -fr $(find /build/v3 -name 'meta.yaml' | grep -v "/latest/" | grep -o ".*/")
+# RUN ./cache_artifacts.sh v3 && chmod -R g+rwX /build
```

Comment:
I think this is mistakenly commented out, there's no commands in `offline-builder` currently.